ghcide/src/Development/IDE/Core/Compile.hs

```diff
@@ -817,7 +850,13 @@ parseFileContents
        -> ExceptT [FileDiagnostic] IO ([FileDiagnostic], ParsedModule)
 parseFileContents env customPreprocessor filename ms = do
    let loc  = mkRealSrcLoc (mkFastString filename) 1 1
-       dflags = ms_hspp_opts ms
+       dflags' = ms_hspp_opts ms
+       dflags =
+         dflags'
+            { staticPlugins =
+                staticPlugins dflags' <> [wingmanMetaprogrammingPlugin]
```

Comment:
@wz1000 is there some way I can hook into ghcide from Wingman to install this plugin, rather than doing it directly in ghcide?

Reply:
Add a new field to `PluginDescriptor` in `hls-plugin-api`.

Reply:
cheers